Controller/Admin/Transaction/TransactionTabPostProcessing.php

```diff
@@ -338,10 +340,14 @@ protected function _getPostProcessingActions()
      */
     private function _filterPostProcessingActions($aPossibleOperations, $oPaymentMethod)
     {
-        if ($oPaymentMethod->getName(true) === SofortPaymentMethod::getName(true)
-            && !$oPaymentMethod->getPostProcessingPaymentMethod(Transaction::ACTION_CREDIT)
-                ->getPayment()->oxpayments__oxactive->value) {
-            return [];
+        $oTransaction = $oPaymentMethod->getPostProcessingTransaction(Transaction::ACTION_CREDIT, $this->oTransaction);
```

Comment:
Why is the ACTION_CREDIT hard coded? Is it not depending on what we are doing?

Reply:
fixed